Lib/idlelib/hyperparser.py

```diff
@@ -23,7 +23,7 @@
 
 
 class HyperParser:
-    def __init__(self, editwin, index):
+    def __init__(self, editwin, index, end_at_eol=True):
```

Comment:
Do you think `wrap` or something like it would work instead of `end_at_eol`?  I had to stop and think about `end_at_eol` to comprehend its meaning.

Reply:
Yes, the name "end_at_eol" isn't great. I'm not sure "wrap" is clear enough though. Perhaps "stop_at_line_end" or "multiline"?